src/app/modules/ix-table/components/ix-table-body/ix-table-body.component.ts

```diff
@@ -50,6 +50,7 @@ export class IxTableBodyComponent<T> implements AfterViewInit {
   readonly columns = input<Column<T, ColumnComponent<T>>[]>();
   readonly dataProvider = input<DataProvider<T>>();
   readonly isLoading = input(false);
+  readonly hasDetailsCards = input(false);
```

Comment:
table-body is a generic table component. I don't think it needs to know about details.
Let's try with something like `templateColumn` instead.